planning/behavior_velocity_planner/src/scene_module/occlusion_spot/grid_utils.cpp

```diff
@@ -80,77 +80,21 @@ void addObjectsToGridMap(const std::vector<PredictedObject> & objs, grid_map::Gr
   }
 }
 
-bool isOcclusionSpotSquare(
```

Comment:
is already alll the occlusion candiate created from eroded occlusion image so no need to find